### ISSUE TITLE
I need help 

### ISSUE THREAD

**Issue by Slushi-Github** [beihu235/Psych-Engine-0.71h-Android#6]
### Describe your problem here.

I wanted to move some things from my engine to this psych engine for android
And even if I manage to compile, it gives me error once I am in android.
Specifically this:
![Screenshot_20230907-040128_Slushi Engine](https://github.com/beihu235/Psych-Engine-0.71h-android/assets/81278332/d5c13c0a-600f-46ed-8da2-fb4fd6e35b4e)

And if I add the file by editing the apk, when I compile and open the app, it does nothing.

I feel that it is a problem of the dependencies that I have to compile.
I have the ones from Psych Engine 0.7.1h, but maybe not the correct ones for android.
Which ones should I have? Since the original Psych Engine provided in Releases does open for me, so it can't be so much my phone's fault. 

### Are you modding a build from source or with Lua?

Source

### What is your build target?

Neko, HashLink, or other build system

### Did you edit anything in this build? If so, mention or summarize your changes.

Several things, but the most important would be the Modcharting Tools (from TheZoroForce240). Since that is what matters to me  Otherwise some text or image added.

**Comment by beihu235** [comment]
look like you change lib.so,it cant change code

**Comment by Slushi-Github** [comment]
> look like you change lib.so,it cant change code

When I tried to compile the game for the first time, I had an error almost at the end because it did not find something from Luajit, and the compiler expected it to be in a path that was not, and I moved the necessary folder, that is, the android one from Luajit to that side
I'm sure it's a problem with the dependencies that I have, but I don't know which ones to use either.
Could you give me that information please?

**Comment by beihu235** [comment]
apk cannot be modified by adding lib.so,you must change source code at github

**Comment by Slushi-Github** [comment]
Yes.. I managed to compile the game through github, but I want to know how to do it locally because it takes a long time

**Comment by beihu235** [comment]
it cant change code at locally,my version only need 6 to 10 mins pack Engine is faster than before

**Comment by beihu235** [comment]
if you want change source code faster,extend hscript load to other state

**Comment by Slushi-Github** [comment]
I don't mean to change, I mean to compile the game, I see that I'm not making my point. 
I want to compile the game from my PC, without relying on Github

**Comment by beihu235** [comment]
Configure the source code environment and package it on your own computer (it's a hassle, but it only takes 30 seconds to package)

**Comment by beihu235** [comment]
I don't have a tutorial on this because I don't even have a computer lol
